playbooks/common/openshift-etcd/config.yml

```diff
@@ -88,6 +88,7 @@
   roles:
   - etcd
   - role: nickhammond.logrotate
+    when: not is_containerized | bool
```

Comment:
I think there was talk about removing the logrotate stuff all together, but I may be mis-remembering